gitops/azure-devops/unit_test/.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: bash
+
+env:
+  - SHUNIT_COLOR='always'
+
+script:
+  # Execute the unit tests.
+  - ./test_runner
+
+os:
```

Comment:
@yradsmikham is this running against Travis CI? Can you include a build badge on the readme?

Reply:
I just tested this locally, and it is not used. It is not running against Travis CI. It also appears that sHUnit2 is not using it. Will remove.